providers/config_default.yaml

```diff
@@ -33,6 +33,10 @@ CLUSTER_API_SERVER_PORT:
 
 CLUSTER_CLASS:
 
+#! ALLOW_LEGACY_CLUSTER will allow legacy based workload cluster creation.
+#! Support values are 'false', 'true'. By default, it's false.
```

Comment:
Support ->Supported. Please also align it to other parameters: do you mean "false", "true"? If so, please update the comment as:  Supported values are "false", "true". By default, it's "false".

Reply:
Changed according to the comment.